src/RepoToolset/tools/VisualStudio.InsertionManifests.targets

```diff
@@ -27,6 +27,7 @@
       <_Args Include="SetupOutputPath=$(VisualStudioSetupInsertionPath)"/>
       <_Args Include="ComponentIntermediateOutputPath=$(VisualStudioSetupIntermediateOutputPath)$(_ComponentName)\"/>
       <_Args Include="SwixBuildPath=$(NuGetPackageRoot)microbuild.plugins.swixbuild\$(MicroBuildPluginsSwixBuildVersion)\"/>
+      <_Args Include="ManifestBuildVersion=$(ManifestBuildVersion)" />
```

Comment:
Consumers of the repo toolset will need to specify this value (see dotnet/project-system#3669) and update their toolset reference.  The change is such that if only one of these changes (the property or the toolset update) is accepted, then nothing will break, but the `.vsman` files just won't have the new `info/buildVersion` property.

Reply:
Why is this not set directly to `VsixVersion`?